packages/costa/src/runtime.ts

```diff
@@ -67,13 +68,13 @@ interface InstallOptions {
   stderr: NodeJS.WritableStream;
 }
 
-function spawnAsync(command: string, args: string[], opts: SpawnOptions, stdio: LogStdio): Promise<void> {
+function spawnAsync(command: string, args: string[], opts: SpawnOptions, stdio?: LogStdio): Promise<void> {
   return new Promise((resolve, reject) => {
     opts.stdio = [ null, 'pipe', 'pipe' ];
     opts.detached = false;
     const child = spawn(command, args, opts);
-    pipeLog(child.stdout, stdio.stdout, stdio.prefix);
-    pipeLog(child.stderr, stdio.stderr, stdio.prefix);
+    stdio?.stdout && pipeLog(child.stdout, stdio.stdout, stdio.prefix);
```

Comment:
I think we could just verify it in the function `pipeLog()`.

Reply:
done